sdk/keyvault/azure-security-keyvault/tests/test_example_keys_async.py

```diff
@@ -0,0 +1,291 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+import time
+import asyncio
+import functools
+
+from azure.core.exceptions import ResourceExistsError, ResourceNotFoundError, HttpResponseError
+from devtools_testutils import ResourceGroupPreparer
+from preparer import VaultClientPreparer
+from test_case import KeyVaultTestCase
+from azure.security.keyvault.aio.vault_client import VaultClient
```

Comment:
`VaultClient` should be exposed such that you can import it from `azure.security.keyvault.aio`.

Reply:
This would also change secrets client import.
Should we update that too then?

Reply:
According to our offline conversation, we would be only updating the key client for now and update the secrets client later to include this change.